src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js

```diff
@@ -232,6 +231,38 @@ define([], function() {
             return [this.options.url, delimiter, this.options.idsParameter, '=', (data || []).join(',')].join('');
         },
 
+        overlayOkCallback: function () {
+            this.sandbox.emit('husky.column-navigation.' + this.options.instanceName + '.get-marked', function (markedCollections) {
+                var toAdd = [],
+                    toRemove = [];
+
+                markedCollections.forEach(function(markedCollection) {
+                    if ($.inArray(markedCollection, this.getData()) < 0) {
+                        toAdd.push(markedCollection);
```

Comment:
Why not simply call `selectLink` here, instead of saving it in another variable?